src/api/lib/tasks/test_webui.rake

```diff
@@ -11,22 +11,46 @@ Rake::TestTask.new do |t|
   t.warning = false
 end
 
+# minitests are a little fragile when it comes to run out of order
+# so we cherry pick some functional tests into group1 that are safe
```

Comment:
How exactly did you determine them to be safe?

Reply:
safe here means they don't require previous jobs and don't interfere with jobs after that. build, source, publish and maintenance tests are a nightmare to parallize

Reply:
> safe here means they don't require previous jobs and don't interfere with jobs after that. 

Please add this to the comment

Reply:
extended the comment